### PR TITLE
fix(mobile): version announcement box shouldn't be shown for non-admins

### PR DESCRIPTION
**Pull request by samip5**

### CODE REVIEW

**Comment by samip5** [comment]
I'm not at all sure about the mobile portion. :D

File: web/src/lib/components/shared-components/version-announcement-box.svelte [hunk 1]
```diff
@@ -24,14 +30,19 @@
 
   onMount(async () => {
     try {
-      githubVersion = await getGithubVersion();
-      if (localStorage.getItem('appVersion') === githubVersion) {
-        // Updated version has already been acknowledged.
-        return;
-      }
+      let isAdmin = await checkIsAdminLoggedIn();
```
Comment (alextran1502) [inline web/src/lib/components/shared-components/version-announcement-box.svelte:33]: You can get the current user info with

```typescript
import { page } from '$app/stores';

const user = $page.data.user;
```

**Comment by martabal** [comment]
Isn't it already the case on the web ? 

https://github.com/immich-app/immich/blob/0abbd85134035b04ec3b980661e9b497d63394e0/web/src/routes/%2Blayout.svelte#L92-L94

**Comment by samip5** [comment]
> Isn't it already the case on the web ?
> 
> https://github.com/immich-app/immich/blob/0abbd85134035b04ec3b980661e9b497d63394e0/web/src/routes/%2Blayout.svelte#L92-L94

To me, that looks like it would only show the server version in the message for non-admins? Granted I'm not sure of that.

**Comment by alextran1502** [comment]
Can you help rebasing the PR please?

**Comment by samip5** [comment]
I have yeeted the web changes, and only thing left are mobile.

**Review by alextran1502 — APPROVED** [review]

**Pull request merged**